src/dos/dosbox/cpp/direct-debugger.cpp

```diff
@@ -67,4 +157,23 @@ extern "C" void EMSCRIPTEN_KEEPALIVE rescanFilesystem() {
   return;
 }
 
+Bitu DosBox_Pause(void) {
+  asyncify_sleep(1);
+  return 0;
+}
+
+extern "C" bool EMSCRIPTEN_KEEPALIVE pauseExecution(bool shouldPause) {
```

Comment:
please add information about this feature to wiki
https://github.com/caiiiycuk/js-dos/blob/gh-pages/v7/docs/dosbox-direct.md

Reply:
Will do.